src/koaApi.js

```diff
@@ -39,11 +39,11 @@ export function setupApp(done) {
   app.use(cors({allowMethods: 'GET,HEAD,PUT,POST,DELETE', credentials: true}))
 
   // Configure Sessions Middleware
-  app.keys = [config.authentication.sessionKey]
+  app.keys = [config.api.sessionKey || 'r8q,+&1LM3)CD*zAGpx1xm{NeQhc;#']
```

Comment:
I think defaulting this might be dangerous. As we also default it in the config. If for some reason the config isn't being read then this exposed key will be used. Can we remove this default value.

Reply:
done:)